cddl/test/FAIL_SoftwareComponent_Measurement_missing.diag

```diff
@@ -27,10 +27,10 @@
             31415161718191a1b1c1d1e1f'
           }
         ],
-      / arm_psa_security_lifecycle / -4501: 12288 / SECURED /,
+      / arm_psa_security_lifecycle / 2395: 12288 / SECURED /,
       / nonce /            10: h'000102030405060708090a0b0c0d0e0f1011
       12131415161718191a1b1c1d1e1f',
-      / arm_psa_partition_id / -4500: -1,
+      / arm_psa_partition_id / 2394: -1,
```

Comment:
Slight inconsistency here between client id (in block definition) & partition id here

Reply:
Yeah, I can't recollect exactly how we ended up here, but we have also this: _`Note that the CDDL label used to be called arm_psa_partition_id.`_ in the Client ID subsection.  I'll raise an issue.